aws_lambda/aws_lambda_stack.py

```diff
@@ -25,6 +25,28 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             table_name="users_table"
         )
 
+        orders_table = aws_dynamodb.Table(
```

Comment:
For clarity we could separate each service into a separate Python function here.

Reply:
you mean a function for creating the tables, a function for creating the lambdas and so on ?

Reply:
No, per service, so the logic is decoupled.

Reply:
Let's do this in another PR, as the other services also use this approach currently.